src/Header/AbstractAccept.php

```diff
@@ -330,9 +331,9 @@ public function match($matchAgainst)
     /**
      * Return a match where all parameters in argument #1 match those in argument #2
      *
-     * @param array $match1
-     * @param array $match2
-     * @return bool|array
+     * @param stdClass $match1
```

Comment:
Rater than `stdClass`, this should be `object`

Reply:
`stdClass` is not a type; it's a default implementation of an object in PHP. I agree with @Ocramius here.